.github/workflows/release.yaml

```diff
@@ -55,6 +55,7 @@ jobs:
           export VERSION=${VERSION} 
           export PREVIOUS_VERSION=${PREVIOUS_VERSION}
           export SKIP_RANGE_LOWER=${SKIP_RANGE_LOWER}
+          git -C operator checkout v${VERSION}
```

Comment:
Can it maybe done in the earlier step of [Checkout code](https://github.com/medik8s/machine-deletion-remediation/pull/125/files#diff-e426ed45842837026e10e66af23d9c7077e89eacbe6958ce7cb991130ad05adaR34-R38)?

Reply:
It could but

1. It needs another token (it can be the same used later)
2. I wasn't able to make the `actions/checkout` `ref` parameter dynamic. It works if I write 

```
- name: Checkout code
  uses: actions/checkout@v3
  with:
      path: operator
      ref: v0.3.0
      fetch-depth: 0
```

but whatever other variable I tried for the tag name, it took it litteraly (e.g. ${VERSION}, ${{ inputs.version }}, etc.)

Reply:
Ok, then no need to further investigate that :D

Reply:
wait, you select the branch / tag in the action's UI, right? It doesn't checkout that version then?

Reply:
![image](https://github.com/medik8s/machine-deletion-remediation/assets/1962082/01e61bbf-c0a1-4e84-b5d7-307ce51b5fde)

Reply:
I missed that selector, thanks!
At the moment I can't test, but let's keep this in hold since it might be unnecessary

![image](https://github.com/medik8s/machine-deletion-remediation/assets/2883614/e19d240a-665b-46bb-8048-b51b77027ef6)